docs/build/isc/v1.1/docs/how-tos/core-contracts/token/erc20-native-token.md

```diff
@@ -35,11 +35,20 @@ NativeTokenID memory id =token.nativeTokenID();
 ### Full Example Code
 
 ```solidity
-function nativeTokenID(uint32 _foundrySN) public view returns (bytes memory) {
-    ERC20NativeTokens token = ERC20NativeTokens(
-        ISC.sandbox.erc20NativeTokensAddress(_foundrySN)
-    );
-    NativeTokenID memory id =token.nativeTokenID();
-    return id.data;
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@iota/iscmagic/ISC.sol";
+
+contract NativeToken {
+
+    function nativeTokenID(uint32 _foundrySN) public view returns (bytes memory) {
+        ERC20NativeTokens token = ERC20NativeTokens(
+            ISC.sandbox.erc20NativeTokensAddress(_foundrySN)
+        );
+        ISC.NativeTokenID memory id = token.nativeTokenID();
```

Comment:
Why did that line change from `NativeTokenID memory id =token.nativeTokenID();`
to ` ISC.NativeTokenID memory id = token.nativeTokenID();` ?